src/libraries/Microsoft.PowerFx.Core/Public/Values/CustomObjectValue.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Text;
+using System.Text.Json;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Public.Types;
+
+namespace Microsoft.PowerFx.Core.Public.Values
+{
+    public interface ICustomObject
+    {
```

Comment:
Should we have IsBoolean/GetBoolean on this as well, or waiting for a future PR?

Reply:
Special constructors will be in a future PR